eng/common/scripts/Verify-RestApiSpecLocation.ps1

```diff
@@ -164,16 +164,6 @@ function Verify-YamlContent([string]$markdownContent, [string]$configFilePath) {
 
 function Verify-PackageVersion() {
   try {
-    $packages = @{}
-    if ($FindArtifactForApiReviewFn -and (Test-Path "Function:$FindArtifactForApiReviewFn")) {
-      $packages = &$FindArtifactForApiReviewFn $ArtifactLocation $PackageName
-    }
-    else {
-      LogError "The function for 'FindArtifactForApiReviewFn' was not found.`
-      Make sure it is present in eng/scripts/Language-Settings.ps1 and referenced in eng/common/scripts/common.ps1.`
-      See https://github.com/Azure/azure-sdk-tools/blob/main/doc/common/common_engsys.md#code-structure"
-      exit 1
-    }
     if (-not $PackageInfoDirectory) {
```

Comment:
Before this was only for local testing but now after your change this will happen every time and I suspect that isn't what we want. The idea was that we would get the already existing packageInfo from the artifacts but it appears that go doesn't publish those. So, the question is should we have go start publishing them? We might need to for other scenarios like APIView. However if we don't do that we shouldn't write the json files to disk we should just call the helper functions that the at save script is calling.

See https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/Save-Package-Properties.ps1#L95

For example there will be cases where looking for `$PackageName.json` will fail that were already handled in that save script https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/Save-Package-Properties.ps1#L115